topics/latex/index.md

```diff
@@ -0,0 +1,13 @@
+---
+aliases: latex-template, latex-class, latex-package, latex-document, latex-examples
+created_by: Leslie Lamport
+display_name: LaTeX
+github_url: https://github.com/latex3
+released: 1985
+short_description: LaTeX is a document preparation system.
+topic: latex
+url: https://www.latex-project.org/
+related: tex
+wikipedia_url: https://en.wikipedia.org/wiki/LaTeX
+---
+LaTeX is a typesetting system used to create technical and scientific documents. It is an alternative to word processing, letting you focus on content over appearance.
```

Comment:
We try to avoid "you" in descriptions. Suggested:

> It is an alternative to word processing, emphasizing focus on content over appearance.

Reply:
Thanks for the suggestion! I only used "you" here because I saw other topics use it, but I'll update the description as soon as I get the chance.

Reply:
@nayafia Before I make the commit, how do you think this sounds?

> It is an alternative to word processing, with **an emphasis** on content over appearance.

or

> It is an alternative to word processing, with **a focus** on content over appearance.

I want to avoid the phrase "emphasizing focus" here if possible. It seems a little repetitive.

Reply:
@GloverDonovan good call. I like `an emphasis` better!

Reply:
Done 🎉